packages/web/components/Landing/OnboardingGame/index.tsx

```diff
@@ -0,0 +1,869 @@
+/* eslint-disable no-nested-ternary */
+import {
+  Box,
+  Button,
+  Icon,
+  keyframes,
+  ListItem,
+  Text,
+  Tooltip,
+  UnorderedList,
+  usePrefersReducedMotion,
+} from '@metafam/ds';
+import externalLinkIcon from 'assets/landing/external-link-icon.png';
+import { useGame } from 'contexts/GameContext';
+import { useOnScreen } from 'lib/hooks/useOnScreen';
+import { get } from 'lib/store';
+import { ReactElement, useCallback, useEffect, useRef, useState } from 'react';
+import { MdDownloading, MdRestartAlt, MdWarning } from 'react-icons/md';
+import {
+  safelyParseContent,
+  safelyParseTextForTyping,
+} from 'utils/stringHelpers';
+
+import { Chiev } from './Chiev';
+import type {
+  GameProperties,
+  GamePropertiesType,
+  IConnection,
+  IElement,
+  IJumper,
+} from './gameTypes';
+import gameJson from './metagame-onboarding-game.json';
+
+export type CurrentElementState = IElement & {
+  elementId: string;
+};
+export type ConnectionStateItem = IConnection & {
+  connectionId: string;
+};
+
+export type CurrentJumperState = IJumper & {
+  jumperId: string;
+};
+
+export const OnboardingGame: React.FC = (): JSX.Element => {
+  const ref = useRef<HTMLDivElement>(null);
+  const scrollContentRef = useRef<HTMLDivElement>(null);
+  const noMotion = usePrefersReducedMotion();
+  const onScreen = useOnScreen(ref);
+  const { gameState, handleChoice, resetGame, visitedElements } = useGame();
+  const [currentElement, setCurrentElement] = useState<CurrentElementState>();
+  const [currentConnections, setCurrentConnections] =
+    useState<ConnectionStateItem[]>();
+  const [currentJumpers, setCurrentJumpers] = useState<CurrentJumperState[]>();
+  const [currentDialogue, setCurrentDialogue] =
+    useState<CurrentSectionDialogueChoices['currentDialogue']>();
+  const [currentChoices, setCurrentChoices] =
+    useState<CurrentSectionDialogueChoices['currentChoices']>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [isTyping, setIsTyping] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const [gameDataState, setGameDataState] = useState<GamePropertiesType | null>(
+    null,
+  );
+  const [welcomeBack, setWelcomeBack] = useState(false);
+  const blink = keyframes`
+    0% {
+      opacity: 0;
+    }
+
+    50% {
+      opacity: 1;
+
+    }
+
+    100% {
+      opacity: 0;
+    }
+  `;
+  const typingAnimation = `${blink} 1s steps(10, start) infinite`;
+  // const pulseAnimation = `${blink} 2s infinite`;
+  const visits = visitedElements();
+  const [chievFound, setChievFound] = useState(false);
+
+  /** Function to async fetch `CONFIG.onboardingGameDataURL` as json and return the data */
+  const fetchGameData = useCallback(async () => {
+    try {
+      setHasError(false);
+
+      // const response = await fetch(CONFIG.onboardingGameDataURL);
+      // console.log('fetchGameData RES', response.status);
+      const data = gameJson as GameProperties;
```

Comment:
Since we're not fetching (same as in `GameContext.tsx`) we don't need this useCallback at all.